src/yb/docdb/docdb_rocksdb_util.cc

```diff
@@ -119,6 +119,25 @@ DEFINE_int32(priority_thread_pool_size, -1,
              "If -1 and max_background_compactions is specified - use max_background_compactions. "
              "If -1 and max_background_compactions is not specified - use sqrt(num_cpus).");
 
+DEFINE_int32(compression_type, 1,
+             "compression_type. "
+             "If enable_ondisk_compression is false - no compression. "
+             "If 0 and enable_ondisk_compression is true - use LZ4. "
+             "If 1 and enable_ondisk_compression is true - use Snappy.");
```

Comment:
can we introduce an enum `YB_DEFINE_ENUM(OnDiskCompression, (kDisabled)(kUseSnappy)(kUseLz4));`

and then remove the `enable_ondisk_compression` flag altogether? and set this flag type to default to `OnDiskCompression::kUseSnappy` by defining it like so:

```
YB_DEFINE_ENUM(OnDiskCompression, (kDisabled)(kUseSnappy)(kUseLz4));

DEFINE_int32(
    compression_type, OnDiskCompression::kUseSnappy, strings::Substitute(
        "On-disk compression to use in RocksDB. Set to $0 to disable compression, $1 to use Snappy, and $2 to use LZ4.",
        OnDiskCompression::kDisabled, OnDiskCompression::kUseSnappy, OnDiskCompression::kUseLz4));
```